dashboard/modules/job/job_manager.py

```diff
@@ -25,6 +27,20 @@
     create_task = asyncio.ensure_future
 
 
+def generate_job_id() -> str:
```

Comment:
we probably want to make this a lil bit more generic if we ever want to reuse the part where we are generating characters

Reply:
unlikely, can do that if it's needed